spec/models/need_spec.rb

```diff
@@ -184,6 +195,30 @@
 
       expect(result.to_a).to eq([new_user, need.user])
     end
+    
+    context "preferred_language_override" do
+      before do
+        need.preferred_language_id = french.id
+        need.office.users << french_user
+        need.office.users << english_user
+      end
+
+      context 'is false' do
```

Comment:
```suggestion
      context 'when it is false' do
```

Similar change below :smile: